zerver/openapi/zulip.yaml

```diff
@@ -6381,12 +6406,37 @@ paths:
             - `oldest`: The oldest message.
             - `first_unread`: The oldest unread message matching the
               query, if any; otherwise, the most recent message.
+            - `date`: Anchors the retrieval to the message closest to the specified date
+              in the `anchor_date` parameter, if any; otherwise, the most recent message.
+
+            **Changes**: `date` enum value is new in Zulip 7.1 (feature level 188).
           schema:
             oneOf:
               - type: string
+                enum: [newest, oldest, first_unread, date]
               - type: integer
           example: 43
           required: true
+        - name: anchor_date
+          in: query
+          description: |
+            Date and time (in ISO 8601 format) to search for messages closest to this datetime.
+            If only the date is provided, the timezone will be interpreted as UTC.
+
+            **Changes**: New in Zulip 7.1 (feature level 188).
```

Comment:
`7.1` is a mistake I'll correct it along with the other changes.

Reply:
yeah, it should just say 8.0, since we don't backport this sort of change to the 7.x style stable releases.